lib/features/family/features/preferred_church/preferred_church_selection_page.dart

```diff
@@ -124,6 +124,11 @@ class _PreferredChurchSelectionPageState
         onTap: () => _onTapConfirm(context),
         analyticsEvent: AnalyticsEvent(
           AmplitudeEvents.preferredChurchSelected,
+          parameters: {
+            'namespace':
```

Comment:
Did you test this? Because I am pretty sure now you're going to get the outOfRange exception again.

Reply:
Its better now! :) (and tested)